test/documentation.spec.ts

```diff
@@ -100,30 +100,35 @@ describe('documentation', () => {
         return parentNames;
       }
 
-      it('has headers for every required sub-option', () => {
-        expect(getConfigSubHeaders('configuration-options.md')).toEqual(
+      it('has headers for every required sub-option', async () => {
+        expect(await getConfigSubHeaders('configuration-options.md')).toEqual(
           getRequiredConfigSubOptions(),
         );
       });
 
-      it.each([...getParentNames()])(
-        '%s has sub-headers sorted alphabetically',
-        (parentName: string) => {
+      it('has sub-headers sorted alphabetically', async () => {
+        const parentNames = getParentNames();
```

Comment:
please use `test.each` again

Reply:
test.each

Reply:
test.each

Reply:
> please use `test.each` again 

 > รับตัวเลือกย่อยที่จำเป็น()
         -
       -
 
- มัน.each([...รับชื่อผู้ปกครอง()])(
- '%s มีส่วนหัวย่อยที่จัดเรียงตามตัวอักษร'
- (parentName: สตริง) => {
+ it('มีส่วนหัวย่อยที่จัดเรียงตามตัวอักษร', async () => {
+ const parentNames = getParentNames();